mne/io/curry/curry.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+import os
+import re
+import numpy as np
+import mne
+
+from ..base import BaseRaw, _check_update_montage
+from ..meas_info import create_info
+from ..utils import _read_segments_file, warn, _find_channels, _create_chs
+from ..constants import FIFF
+
+INFO_FILE_EXTENSION = {7: '.dap', 8: '.cdt.dpa'}
+LABEL_FILE_EXTENSION = {7: '.rs3', 8: '.cdt.dpa'}
+EVENT_FILE_EXTENSION = {7: ['.cef', '.ceo'], 8: ['.cdt.cef', '.cdt.ceo']}
```

Comment:
Add `# XXX: currently there is no .ceo file the test suit`

Reply:
yes, i will have to make a bigger change concerning the test files.
But I've used curry only very occassionally and need to check whether i have access to it, in order to create new files.
So please excuse me if these test problems take a while!

Reply:
Ok, so i can't get access to curry at my department, which means i can't supply .ceo testing data.
Should we just cut the support for this kind of file?

Reply:
I'm fine with supporting a subset at first. We can always add it later